.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -0,0 +1,124 @@
+name: Code for Africa | Deploy | REVIEW APP
+
+on:
+  pull_request:
+    paths:
+      - "apps/codeforafrica/**"
+      - "Dockerfile.codeforafrica"
+      - ".github"
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.ref }}"
+  cancel-in-progress: false
+
+env:
+  DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://azureuser@ui-1.dev.codeforafrica.org"
+  IMAGE_NAME: "codeforafrica/codeforafrica-ui"
+  ORIGINAL_APP_NAME: "codeforafrica-ui"
+  NEXT_PUBLIC_APP_URL: "http://codeforafrica-ui-pr-${{github.event.pull_request.number}}.cms.review.codeforafrica.org"
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  APP_NAME: codeforafrica-ui-pr-${{ github.event.pull_request.number }}
+
+jobs:
+  review_app:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [18]
+        os: [ubuntu-latest]
+    if: github.event_name == 'pull_request' && github.event.action != 'closed'
+    steps:
+      - name: Cloning repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          path: /tmp/.buildx-cache
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+
+      - name: Clone Database
+        uses: Maggi64/mongo-clone-action@0.3
+        with:
+          source: ${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ secrets.CODEFORAFRICA_MONGO_DB_NAME }}
+          target: ${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ env.APP_NAME }}
```

Comment:
We can start with this but I'll be much, much happier if we move away from these unofficial actions. We can have `dokku-pr-create-db-mongo` trigger that listen to `post-create` and _create_ the mongo db all from with the dev instance.